lms/djangoapps/verify_student/models.py

```diff
@@ -912,10 +913,11 @@ def send_request(self, copy_id_photo_from=None):
             return fake_response
 
         headers, body = self.create_request(copy_id_photo_from=copy_id_photo_from)
+
         response = requests.post(
             settings.VERIFY_STUDENT["SOFTWARE_SECURE"]["API_URL"],
             headers=headers,
-            data=json.dumps(body, indent=2, sort_keys=True, ensure_ascii=False).encode('utf-8'),
+            data=simplejson.dumps(body, indent=2, sort_keys=True, ensure_ascii=False).encode('utf-8'),
```

Comment:
Why change to simplejson here?  It should be the same as JSON right?

Reply:
yeah its same but json is not very frequently updated as simple json. In this case when the body consists bytes json is unable to dump that. But simplejson can dump those.

Reply:
Why not just say `body.decode('utf-8')` before passing it to `json.dumps()`

Reply:
bcz its a dict and children have bytes so we have to decode all the children.